scripts/ci_mingw64_geany.sh

```diff
@@ -145,11 +145,17 @@ patch_version_information() {
 		MAJOR="${BASH_REMATCH[1]}"
 		MINOR="${BASH_REMATCH[2]}"
 		PATCH="${BASH_REMATCH[4]}"
-		if [ -z "${PATCH}" ] || [ "${PATCH}" = "0" ]; then
-			MINOR="$((MINOR-1))"
-			PATCH="90"
+		if [[ "${MINOR}" = "0" && (-z "${PATCH}" || "${PATCH}" = "0") ]]; then
+			MAJOR="$((MAJOR-1))"
+			MINOR="99"
+			PATCH="99"
 		else
-			PATCH="$((PATCH-1))"
+			if [[ -z "${PATCH}" || "${PATCH}" = "0" ]]; then
+				MINOR="$((MINOR-1))"
+				PATCH="99"
+			else
+				PATCH="$((PATCH-1))"
+			fi
 		fi
```

Comment:
Isn't this a simpler version of the exact same thing?
```bash

		if [ -z "${PATCH}" ] || [ "${PATCH}" = "0" ]; then
			if [ "${MINOR}" = "0" ]; then
				MAJOR="$((MAJOR-1))"
				MINOR="99"
			else
				MINOR="$((MINOR-1))"
			fi
			PATCH="99"
		else
			PATCH="$((PATCH-1))"
		fi
```

Reply:
Sweet, thank you.